src/app/shared/routing/group-route.ts

```diff
@@ -7,8 +7,13 @@ export interface GroupRoute extends ContentRoute {
 /**
  * Return a url array (`commands` array) to the given group, on the given page.
  */
-export function urlArrayForGroupRoute(route: GroupRoute, page: 'edit'|'details' = 'details'): any[] {
+export function urlArrayForGroupRoute(route: GroupRoute, page: 'edit'|'details'|'users' = 'details'): any[] {
```

Comment:
The page parameter is supposed to be a group page... "users" is no a subpage of group.

Reply:
Fixed